Classes/git/PBGitRepository.m

```diff
@@ -688,6 +688,77 @@ - (NSString *) infoForRemote:(NSString *)remoteName
 	return output;
 }
 
+- (NSDictionary *) remoteURLs
+{
+	int retValue = 1;
+	NSString *output = [self outputInWorkdirForArguments:@[@"remote", @"-v"] retValue:&retValue];
+	
+	if (retValue)
+		return nil;
+
+	NSArray* lines = [output componentsSeparatedByString:@"\n"];
+	
+	NSMutableDictionary* urls = [[NSMutableDictionary alloc] init];
+	for (NSString* line in lines) {
+		NSArray* split = [line componentsSeparatedByString:@"\t"];
+		NSString* remoteName = split[0];
+		
+		if (split.count < 2) {
+			continue;
+		}
+		
+		NSString* urlString = [split[1] componentsSeparatedByString:@" "][0];
+
+		NSURL* url = [[NSURL alloc] initWithString:urlString];
+		
+		if (!url) {
+			continue;
+		}
+		
+		[urls setObject:url forKey:remoteName];
+	}
+	
+	return urls;
+}
+
+- (void) openWebsiteOfRemote:(NSString*)remoteName {
+	NSDictionary* remotes = self.remoteURLs;
+	
+	NSURL* remoteURL = remotes[remoteName];
+	NSString* path = remoteURL.absoluteString;
+	
+	if ([path containsString:@"github.com"]) {
+		[[NSWorkspace sharedWorkspace] openURL:remoteURL];
```

Comment:
Will this work for SSH URLs like git@github.com:gitx/gitx.git?

It would be good to have a method that takes a git URL and returns a website URL, then you could easily write unit tests for URLs of different forms.